### PR TITLE
Created Project

### PR DESCRIPTION
**Pull request by larson-carter**
I created the ruby project in Ruby Mine - JetBrains.

I also created a few ruby files for where we will work inside of.

Implement basic getRepologyAPIJSON
Implement basic getHomebrewAPIJSON

Get prettier setup for Repology and HomeBrew.

### CODE REVIEW

**Comment by zencephalon** [comment]
I'm thinking U probably want a `.gitignore` with the `.idea/` folder in it.

**Comment by BaffourAdu** [comment]
@larson-carter I think this pull request has grown beyond the initial description provided can you please update that as well as the title of the pull request?

File: getRepologyJson.rb [hunk 1]
```diff
@@ -1 +1,9 @@
+require 'net/http'
+require 'json'
 
+url = 'https://repology.org/api/v1/projects/?inrepo=homebrew&outdated=1'
+uri = URI(url)
+response = Net::HTTP.get(uri)
+parsed = JSON.parse(response)
+
+File.write("repology-JSON/unformatted/latest.json", parsed)
```
Comment (BaffourAdu) [inline getRepologyJson.rb:9]: Before writing to the file I will suggest checking if the file or directory already exists. We also discussed making this file named with a date-stamped right?

Comment (larson-carter) [reply getRepologyJson.rb:9]: Yep, right now I'm just wanting to get the data to be stored. Once I can get the data stored properly I will implement the feature to store the .json with the proper format. Or would you like to do so?

**Comment by larson-carter** [comment]
> @larson-carter I think this pull request has grown beyond the initial description provided can you please update that as well as the title of the pull request?

I have updated it.

File: getRepologyJson.rb (hunk 1 above)
Comment (BaffourAdu) [reply getRepologyJson.rb:9]: I am currently looking into it and don't have solid answers yet but from my experience, with that many lines of data is better to store it in a txt file taking advantage of line breaks and streams

Comment (BaffourAdu) [reply getRepologyJson.rb:9]: @larson-carter I have been able to check and create a directory, as well as create a date stamp file. I would push my branch so you can check my updates

Comment (larson-carter) [reply getRepologyJson.rb:9]: Amazing. Let me know on discord when it is pushed!

Comment (awasum) [reply getRepologyJson.rb]: Is this conversation resolved?

**Comment by awasum** [comment]
I changed the status to ready for review so we can check and merge ASAP. Good work @larson-carter and @BaffourAdu

**Comment by awasum** [comment]
> Maybe U want a `.gitignore` with the `.idea/` folder in it.

@larson-carter were you able to ignore the .idea/ folder for generated by Rubymine?

File: .gitignore [hunk 2]
```diff
@@ -9,6 +9,8 @@
 /test/tmp/
 /test/version_tmp/
 /tmp/
+/data
+.DS_Store
```
Comment (BaffourAdu) [inline .gitignore:14]: We need to add `.idea/` as @zencephalon mentioned earlier in the conversation

Comment (larson-carter) [reply .gitignore:14]: I will fix this right now!

**Comment by larson-carter** [comment]
> Add .idea/ folder in .gitignore file and squash and force push to this branch if needed. So the idea/ folder is not part of the project.

Fixed.

**Comment by awasum** [comment]
> > Add .idea/ folder in .gitignore file and squash and force push to this branch if needed. So the idea/ folder is not part of the project.
> 
> Fixed.

THanks...just saw that now...

**Pull request merged**